truffle/com.oracle.truffle.api.instrumentation/src/com/oracle/truffle/api/instrumentation/InstrumentationHandler.java

```diff
@@ -536,17 +538,18 @@ protected void visitInstrumented(Node node, SourceSection section) {
     }
 
     /**
-     * Instrumenter implementation for {@link TruffleInstrument}.
+     * Instrumenter implementation for use by {@linkplain TruffleInstrument external clients} of
+     * instrumentation.
      */
-    final class InstrumentationInstrumenter extends AbstractInstrumenter {
+    final class ClientInstrumenter extends AbstractInstrumenter {
```

Comment:
If you think InstrumentClientInstrumenter is better than InstrumentInstrumenter so feel free to change it. I have no strong opinion. I also had no problem with InstrumentInstrumenter, given its an internal class.